_posts/2016-10-10-visualise-interruptions-with-a-lego-wall.md

```diff
@@ -0,0 +1,119 @@
+
+Team members concerns turn into frustrations and become internalised. Impediments become even harder to identify when people are afraid to raise their voice in front of stakeholders.
+
+##The LEGO Wall
+
+{% img /assets/img/custom/blog/2016-10-10-visualise-interruptions-with-a-LEGO-wall/lego-wall-overview.jpg"" %}
+
+The LEGO Wall has been conceived to reduce the fear factor when identifying impediments. It is an artifact to keep interruptions visible while ensuring anonymity.
+
+The whole Scrum Team (PO included!) is involved and the Scrum Master facilitates the LEGO Wall Setup Session, explaining the importance of tracking interruptions. The LEGO Wall gives voice to the team’s frustrations. By collecting and displaying real data, it creates the conditions to discuss the interruptions. At a glance the team is able understand:
+
+ - How much time they spend working on delivering the value agreed during the Sprint planning
+ - How much time they spend working on unplanned activities
+ 
+ The core part of the Setup Session is to build a collective understanding of the type of interruptions the team has to deal with. The teammates share their experience and at the end of the discussion they decide to keep track of top categories of interruptions – keeping them to a maximum of five. After that, they can assign a brick colour to each of them.
+
+In the example shown in the pictures the team identified 4 categories:
+
+ - Production Support: <label style="color: #10D11A"> GREEN </label> 
+ - Release Support: <label style="color: #FFC707"> YELLOW </label> 
+ - Business as Usual: <label style="color: #FF060E"> RED </label> 
+ - Help the PO: <label style="color: #2065FF"> BLUE </label> 
+
+“Business as Usual (BAU)” is the time spent on the ceremonies and the artifacts associated with the development process. These include: meetings, creating reports, writing documents, etc.
+
+The category “Help the PO” encompasses the activities that the development team is asked to do to help the PO prioritise stories (backlog refinement sessions). In fact these activities should be considered part of the work, but too often the development team perceives this activities as interruptions. Work is not only about coding, ux design or testing; it should encompass collaboration with the PO because these activities are essential to delivering value.
+
+
+##Materials and preparation
+
+{% img /assets/img/custom/blog/2016-10-10-visualise-interruptions-with-a-LEGO-wall/lego-wall-materials.jpg "" %}
+
+You can build a LEGO Wall with as many **base plates** as you need, in order to:
+
+ - Display the days of the Sprint (2 boards are enough to keep track of a 2-week Sprint)
+ - Display as many rows as team members (4 developers as shown in the photos)
+ - A legend of the colours used (4 colours + white)
+
+You will need **coloured LEGO bricks** to display interruptions and **LEGO figures** to represent an anonymous team member.

```

Comment:
I think we need to replace `coloured` with `colourful` as it is mostly used in a racist context